docs/docs/building-modules/09-module-interfaces-continued.md

```diff
@@ -0,0 +1,158 @@
+# Module Interfaces Continued
+
+:::note Synopsis
+This document details how to build CLI and REST interfaces for a module. Examples from various Cosmos SDK modules are included.
+:::
+
+:::note
+
+### Pre-requisite Readings
+
+* [Building Modules Intro](./01-intro.md)
+
+:::
+
+## AutoCLI
+
+The `autocli` package is a Go library for generating CLI (command line interface) interfaces for Cosmos SDK-based applications. It provides a simple way to add CLI commands to your application by generating them automatically based on your gRPC service definitions. Autocli generates CLI commands and flags directly from your Protocol Buffer messages, including options, input parameters, and output parameters. This means that you can easily add a CLI interface to your application without having to manually create and manage commands.
+
+### Getting Started
+
+Here are the steps to use the `autocli` package:
+
+1.  Define your app's modules that implement the `appmodule.AppModule` interface.
+2.  Create an instance of the `autocli.AppOptions` struct that specifies the modules you defined.
+3.  Use the `RootCmd()` method provided by `autocli` to generate a root command that includes all the CLI commands for the specified modules.
```

Comment:
Are we expecting apps to use that command for their modules CLI?
I thought the way to go was to use `EnhanceRootCommand` directly in the application itself on the root command, so that it enriches the command. Can we explain that way under `building-apps` and link it here?
This works too, but we should precise again where it these files are usually found (`client/cli`)